web/src/index.ejs

```diff
@@ -3,13 +3,21 @@
 <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title><%= config.appName %></title>
+    <script type="application/ld+json">
+        {
+            "@context" : "https://schema.org",
+            "@type" : "WebSite",
+            "name" : "<%= config.appName %>",
+            "url" : "https://<%= config.hostName %>"
+        }
+    </script>
 
     <!-- Nginx Server Side Include template for dynamic social media previews -->
     <!--# if expr="$render_title = yes" -->
     <!--# include virtual="/proxy/socialmeta/$request_uri" -->
     <!--# else -->
     <!-- Default tags for a prettier social media preview, will be overwritten in Helmet.tsx for most pages. -->
-    <title><%= config.appName %></title>
```

Comment:
@svenseeberg what is the background behind this whole if/else comments and so on? Is it okay to move the `<title>` tag out of it?

Reply:
For [certain user agents](https://git.tuerantuer.org/DF/salt/src/commit/2418c8a97ffd416ddf0ce2d6dfb38bb97b2976e7/states/webapp/files/nginx.conf#L14) our server renders the [content of the page](https://git.tuerantuer.org/Integreat/social-media-meta-tags/src/branch/master/meta.php)
To test this you can manually set your User Agent to e.g. "Twitterbot 1" and test this. Although I see that it is currently not working properly -.-
Therefore having both jsonld and the title could break this behavior. Not sure though.